assets/dev/js/frontend/utils/lightbox/lightbox.js

```diff
@@ -358,30 +399,80 @@ module.exports = elementorModules.ViewModule.extend( {
 		}
 
 		if ( showZoom ) {
-			elements.$iconZoom = $( '<i>', {
-				class: slideshowClasses.iconZoomIn,
-				role: 'switch',
-				'aria-checked': false,
-				'aria-label': i18n.zoom,
-			} );
+			const iconElement = elementorFrontend.config.experimentalFeatures.e_font_icon_svg ? zoomInBold.element : '<i>',
+				showZoomElements = [],
+				showZoomAttrs = {
+					role: 'switch',
+					'aria-checked': false,
+					'aria-label': i18n.zoom,
+				},
+				zoomAttrs = {
+					...showZoomAttrs,
+				};
+
+			if ( ! elementorFrontend.config.experimentalFeatures.e_font_icon_svg ) {
+				zoomAttrs.class = slideshowClasses.iconZoomIn;
+			}
+
+			elements.$iconZoom = $( iconElement ).attr( zoomAttrs ).on( 'click', this.toggleZoomMode );
+
+			showZoomElements.push( elements.$iconZoom );
+
+			if ( elementorFrontend.config.experimentalFeatures.e_font_icon_svg ) {
+				const zoomOutAttrs = {
+					...showZoomAttrs,
+					class: classes.hidden,
+				};
+
+				elements.$iconZoomOut = $( zoomOutBold.element ).attr( zoomOutAttrs ).on( 'click', this.toggleZoomMode );
 
-			elements.$iconZoom.on( 'click', this.toggleZoomMode );
+				showZoomElements.push( elements.$iconZoomOut );
+			}
 
-			elements.$header.append( elements.$iconZoom );
+			elements.$header.append( showZoomElements );
 
-			this.$buttons = this.$buttons.add( elements.$iconZoom );
+			this.$buttons = this.$buttons.add( showZoomElements );
 		}
 
 		if ( showFullscreen ) {
-			elements.$iconExpand = $( '<i>', {
-				class: slideshowClasses.iconExpand,
-				role: 'switch',
-				'aria-checked': false,
-				'aria-label': i18n.fullscreen,
-			} ).append( $( '<span>' ), $( '<span>' ) );
-			elements.$iconExpand.on( 'click', this.toggleFullscreen );
-			elements.$header.append( elements.$iconExpand );
-			this.$buttons = this.$buttons.add( elements.$iconExpand );
+			const iconElement = elementorFrontend.config.experimentalFeatures.e_font_icon_svg ? frameExpand.element : '<i>',
+				fullScreenElements = [],
+				fullScreenAttrs = {
+					role: 'switch',
+					'aria-checked': false,
+					'aria-label': i18n.fullscreen,
+				},
+				expandAttrs = {
+					...fullScreenAttrs,
+				};
+
+			// Only if the experiment is not active, we use the class-name in order to render the icon.
+			if ( ! elementorFrontend.config.experimentalFeatures.e_font_icon_svg ) {
```

Comment:
Add a class property for `e_font_icon_svg`

Reply:
Done.